assets/admin/editor-wizard/steps/course-upgrade-step.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import useSenseiProExtension from '../../../extensions/use-sensei-pro-extension';
```

Comment:
We should add another comment here to Internal dependencies.

Reply:
Fixed in [c5e62a7](https://github.com/Automattic/sensei/pull/5169/commits/c5e62a7ca289f1f11b309d4af7e11da4a7f231e8). Thanks!